### DISCUSSION TITLE
The generated PDF is incorrect when including latex code block with comment

### DISCUSSION THREAD

**Discussion by zousiyu1995** [Witiko/markdown#346] [Q&A] [answered]
The minimum working example is as follows,

`md` file is,
![image](https://github.com/Witiko/markdown/assets/7608449/0a1c3da2-c2cd-4b39-8521-f3e034415e17)

`latex` file is,
```latex
\documentclass{article}

\usepackage{xcolor}
\usepackage{minted}
    \definecolor{mintbg}{rgb}{0.95,0.95,0.95}
    \setminted{%
        bgcolor=mintbg,
        baselinestretch=1,
        breaklines=true,
        breakanywhere=true,
    }
\usepackage{markdown}
    \markdownSetup{
        hybrid=true,
        fencedCode=true,
    }

\begin{document}

\markdownInput{test.md}

\end{document}
```

The PDF file is, 
![image](https://github.com/Witiko/markdown/assets/7608449/30421f56-59ad-473a-b27f-930d58fac121)

The `## heading2` should display in a heading style. However, it is incorrectly displayed in a code block style. This bug was probably caused by the `% comment` because I was able to get the correct layout after removing it.

**Comment by zousiyu1995** [comment, marked as answer]
I made a mistake. When `hybrid=true`, `%` will be interpreted as a comment mark.

**Reply by Witiko** [reply]
That is correct. You need to either disable `hybrid=true` or write `\%` instead of `%`.

Disabling `hybrid=true` is encouraged, see https://github.com/Witiko/markdown/discussions/302#discussioncomment-5902455 and https://youtu.be/U8XjTOhJkg0?t=10m56s for more principled ways of mixing markdown with TeX.
